core/src/main/java/bio/terra/pearl/core/service/notification/substitutors/EnrolleeEmailSubstitutor.java

```diff
@@ -45,6 +46,13 @@ protected EnrolleeEmailSubstitutor(EnrolleeContext ruleData, NotificationContext
         valueMap.put("siteMediaBaseUrl", getImageBaseUrl(contextInfo.portalEnv(), contextInfo.portalEnvConfig(), contextInfo.portal().getShortcode()));
         valueMap.put("siteImageBaseUrl", getImageBaseUrl(contextInfo.portalEnv(), contextInfo.portalEnvConfig(), contextInfo.portal().getShortcode()));
         valueMap.put("profile", enrolleeContext.getProfile());
+        if (Objects.nonNull(enrolleeContext.getParticipantUser())
+                && StringUtils.contains(enrolleeContext.getParticipantUser().getUsername(), "-prox-")) {
```

Comment:
we'll want to make this more robust -- this might not work in the context of age-up protocols, there may be a period of time when they have their real email address as their account name but are still proxied. 

I'd add an "isGoverned" field to EnrolleeContext, and a query in EnrolleeContextService to populate it

Reply:
I found it simpler to just fetch all the enrollee relations and populate them - perhaps a heavyweight solution, though